crawl-ref/source/acquire.cc

```diff
@@ -549,7 +549,6 @@ static int _acquirement_missile_subtype(bool /*divine*/, int & /*quantity*/)
 
     switch (skill)
     {
-    case SK_SLINGS:    result = MI_SLING_BULLET; break;
     case SK_BOWS:      result = MI_ARROW; break;
```

Comment:
You need to replace this with `result = MI_STONE` rather than removing the case altogether.